quickstart.md

```diff
@@ -100,30 +100,23 @@ There are two layers of security available before the charger has the opportunit
 
 Once a charger has a Charging Station entry and its password has been set, you can connect it to the security profile 1 websocket server at `ws://localhost:8082`.
 
-If you want to add a password for security profile 1, send the following request to the CitrineOS API.
+If you want to add a password for security profile 1 and 2, send the following request to the CitrineOS API.
 
 ```shell
-curl --location --request PUT 'localhost:8080/data/monitoring/variableAttribute?stationId=cp001&setOnCharger=true' \
+curl --location --request POST 'localhost:8080/data/configuration/password?callbackUrl=csms.pro/api/notifications' \
 --header 'Content-Type: application/json' \
 --data '{
```

Comment:
Should we keep the VariableAttribute API here as well, or is it fine to remove it?

Reply:
It's fine to remove it!